docs/contributors_agreement_explained.rst

```diff
@@ -8,8 +7,7 @@ which assigns copyright in the code to the Plone Foundation, the non-profit orga
 
 This document explains the purposes of this, along with questions and answers about what this means.
 
-The Plone Contributor Agreement can be found at:
-`<https://plone.org/foundation/contributors-agreement/agreement.pdf>`_.
+You can download *The Plone Contributor Agreement* from `plone.org <https://plone.org/foundation/contributors-agreement/agreement.pdf>`_.
```

Comment:
I think this link should point instead to https://plone.org/foundation/contributors-agreement/ rather than directly to the PDF, and the information here on docs.plone.org about the contributor agreement should not be duplicated here; it is best left on plone.org